README.md

```diff
@@ -1,10 +1,13 @@
+Unofficial fork of ulsdevteam/ojs-plum-plugin. Upgraded/maintained for OJS 3.4/3.5 by Saddam Al-Slfi.
```

Comment:
I think this should also be removed from this PR as it only applies to the fork.